

![EvoMol-Lab](EvoMol.png)

## 

## EvoMol-Lab

The Laboratory of Evolution of Molecules and Systems (EvoMol-Lab) is one of the research groups associated with the [Bioinformatics Multidisciplinary Environment (BioME)](https://bioinfo.imd.ufrn.br), at the [Metrópole Digital Institute (IMD)](https://imd.ufrn.br) of the Federal University of Rio Grande do Norte ([UFRN](https://ufrn.br)).

EvoMol-Lab's mission is to study evolutionary processes that act on molecules, supramolecular complexes, and systems (pathways and interaction networks) and the structural and functional effects caused by genetic variation, using approaches from Bioinformatics, molecular modeling, and evolutionary sequence analysis.

### PT-Br

O Laboratório de Evolução de Moléculas e Sistemas (EvoMol-Lab) é um dos grupos de pesquisa associados ao [Centro Multiusuário de Bioinformática (BioME)](http://biome.ufrn.br), do [Instituto Metrópole Digital (IMD)](http://imd.ufrn.br) da Universidade Federal do Rio Grande do Norte ([UFRN](http://ufrn.br)). O EvoMol-Lab tem como missão estudar processos evolutivos que atuam em moléculas, complexos supramoleculares e sistemas (vias e redes de interação) e os efeitos estruturais e funcionais provocados pela variação genética, utilizando abordagens da Bioinformática, modelagem molecular e análise evolucionária de sequências. Seu Principal Investigador é o Prof. João Paulo MS Lima, Professor Titular da UFRN.

## Our tutorials page:

[Tutoriais BioinfoX](https://jpmslima.github.io)

## Our tools:

"EvoMol-Lab has a series of ***"magic"*** tools to help you with your work and analyses.

 ---

 <img src="CaRinDB.png" alt="CarinDB" width="100" />

### CaRinDB

CaRinDB is an interactive database designed to streamline cancer mutation research by integrating data from The Cancer Genome Atlas (TCGA) and advanced structural analysis tools, along with advanced effect predictions and molecular features such as Residue Interaction Networks (RINs) derived from Protein Data Bank experimental structures and AlphaFoldDB computational models. Covering 33 distinct cancer types, CaRinDB offers a broad spectrum of insights into cancer mutation dynamics.

[CaRinDB](https://bioinfo.imd.ufrn.br/CaRinDB/)

---
<img src="SlytheRINs.png" alt="SlytheRINs" width="100" />

### SlyTheRINs:

SlytheRINs is an interactive Python web application built with Streamlit (https://streamlit.io), designed to analyze and compare protein Residue Interaction Networks (RINs). These graphs represent the amino acid residues as the nodes and their chemical interactions as the edges. They can be easily generated from a single or multiple protein structures, using computational modeling methods, normal-based mode, or molecular dynamics simulation. SlytheRINs provides an analysis dashboard to explore and compare network parameters, identify significant residue-level changes across different conformations, analyze chemical interaction types, and integrate these findings with predicted mutation effects from AlphaMissense (in the case of human proteins). 

[SlyTheRins](https://slytherins.streamlit.app)

---
<img src="RoC-Logo.png" alt="RoC" width="100" />

### The Room of Conformations (RoC)

A user-friendly desktop tool for generating all-atom protein conformational ensembles using Normal Mode Analysis.

[The Room of Conformations](https://github.com/evomol-lab/RoC)

[RoC - GoogleColab Version](https://colab.research.google.com/drive/1jd3qgAZPF9bWxlcCjYURpFQAEYW102Y5?usp=sharing)

---

<img src="RevelioPlots.png" alt="RevelioPlots" width="100" />

### RevelioPlots

RevelioPlots is an interactive web application designed for the quality assessment of protein structure models. By parsing Crystallographic Information Files (.cif or .mmcif), the app provides a suite of intuitive visualizations centered around the per-residue Local Distance Difference Test (pLDDT) score, a key metric for model confidence, particularly in structures generated by tools like AlphaFold.

#### APP:
[RevelioPlots](https://revelioplots.streamlit.app).
#### GitHub:
[RevelioPlots Github](https://github.com/jpmslima/RevelioPlots)

---

<img src="HufflePlots.jpg" alt="HufflePlots" width="100" />

### HufflePlots

ufflePlots is a simple and interactive tool to visualize, analyze and compare RMSD and RMSF plots of protein structural trajectories data from Molecular Dynamics or Normal mode-based geometric simulations.

[HufflePlots](https://protplots.streamlit.app).

---

<img src="DDEA.png" alt="DDEA" width="100" />

### Differential Expression Diagonal Alley (DDEA)

DDEA is a simple Streamlit application to analyze Case x Control gene expression output data from GEO. Allows users to upload a .tsv file, a custom gene list (optional), and specify a comparison string, with controls in a sidebar. DDEA then plots differentially expressed genes using Plotly and displays their p-values.

[DDEA](https://ddealley.streamlit.app/)

---

<img src="Rictusempra.png" alt="Rictusempra" width="100" />

### *Rictusempra*

URictusempra is a web-based cheminformatics tool for interactively visualizing small molecules and calculating their most likely protonation state at a given physiological pH. It provides a simple interface to generate 2D and 3D molecular structures and prepare them for further computational chemistry tasks like molecular docking or simulation.

[Rictusempra](https://rictusempra.streamlit.app)

---

<img src="MaraudersGenoMap.png" alt="MaraudersGenoMap" width="100" />

### Marauder's GenoMap

A comprehensive pipeline for the discovery of protein families in de novo assembled genomes and transcriptomes.

[Marauder's GenoMap](https://github.com/evomol-lab/MaraudersGenoMap)

--- 

<img src="PensievePlotter.png" alt="PensievePlotter" width="100" />


### The Pensieve Plotter

An interactive tool for visualizing Extended Bayesian Skyline Plots (EBSP) from BEAST log files.

[The Pensieve Plotter](https://pensieveplotter.streamlit.app/)
